mozilla/foundation.mozilla.org
# removed intro_text field from buyersguide

# Description



Link to sample test page:
Related PRs/issues: #9612

Per request of the design team, this PR removes the PNI homepage's unused `intro text` field, and any references to it in other files.

# Steps to test

1. Check this branch out and run either `inv new-env` or `inv migrate`
2. Visit the PNI homepage in the CMS
3. Note that the `intro_text` field is no longer there
4. If everything is looking as expected, testing is complete!

# Screenshots

Before:
![image](https://user-images.githubusercontent.com/18314510/201817972-529947ac-84ab-4f46-9b92-ccaa1418a39a.png)

After:
![Screenshot 2022-11-14 at 19-16-23 Wagtail - Editing Buyers Guide Page Privacy not included](https://user-images.githubusercontent.com/18314510/201818103-0ddedc46-39b5-4446-b992-c5ff061ce593.png)


# Checklist



**Tests**

**Changes in Models:**

**Documentation:**
